index.html

```diff
@@ -1,31 +1,72 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Awesome Books</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <div class="logo">Awesome Books</div>
+        <ul class="nav-items">
+          <li class="nav-links"><a href="#list" class="link-1">Add new</a></li>
+          <li class="nav-links">
+            <a href="#form-field" class="link-2">Booklist</a>
```

Comment:
- [ ] [REQUIRED] Please keep the main design layout, according to which the navigation elements should be placed in the following form: 1: `Book List`; 2: `Add New`; 3: `Contact`; Please change the place `"Add new"` item and `"Book List"` like a picture below:

![image](https://user-images.githubusercontent.com/63915024/219047782-6cebc40e-a122-4b79-8c7c-bac64759c165.png)